README.md

```diff
@@ -7,71 +7,120 @@
 
 ## System Requirements
 
+In order to run the teams application locally so you can contribute, 
+you will need to have the following dependencies already installed. 
+Please see the [wiki](https://github.com/rails-girls-summer-of-code/rgsoc-teams/wiki) 
+for more info on installing these requirements. 
+
 * PostgreSQL 9.3 or newer
 * Ruby 2.4.1
 
-### Setup on Ubuntu
+Instructions are given for Ubuntu or macOS. If you use macOS, we also 
+strongly recommending installing/using [Homebrew](https://brew.sh/) for 
+managing dependencies. 
+
+# Contributing
+
+Thanks so much for your contribution! To get setup to develop the RGSoC 
+teams app, please follow the steps below. Here's some handy links within 
+this section of the README:
+ * [**Quick Start**](#quick-start) - gives a high-level overview 
+of the required steps
+ * [**Detailed Instructions**](#detailed-instructions) - gives 
+this same information, but includes more notes about the individual 
+commands and prompts to expect 
+* [**Optional Configs**](#optional-configs) - gives instructions for
+setting up other tools, such as Google Places API and Mailtrap which 
+may be helpful 
+
+### Quick Start
+0. [Setup System Requirements](#setup-system-requirements)
+1. [Configure Database](#configure-database) - Create a postgres user with superuser rights called `rgsoc` 
+with password `rgsoc`. Setup your `config/database.yml` per the example:
+`config/database.yml.example`
+2. [Other dependencies & Sample Data](#other-dependencies--sample-data) - install with bundler, configure other services
+3. [Run the app](#run-the-app) - run your rails server and console. The application will 
+launch at [`http://localhost:3000/`](http://localhost:3000/).
+
+## Detailed Instructions
+
+### 0. Setup System Requirements
+
+**Setup on Ubuntu**
 ```bash
 # Install required packages
 $ sudo apt-get install postgresql libpq-dev libcurl3 libcurl3-gnutls libcurl4-openssl-dev postgresql-contrib-9.3
-# Create database user rgsoc with password rgsoc
-$ sudo -u postgres createuser -P -s rgsoc
-Enter password for new role: rgsoc
-Enter it again: rgsoc
 ```
-
-### Setup on OS X
+**Setup on OS X**
 ```bash
 # Install required packages
 $ brew install ruby
 $ gem install bundler
 $ brew install postgres
-# Make sure to follow the instructions printed on the screen for postgres
+```
+
+#### 1. Configure Database
+
+1. Create your database (if you have a new install of postgres)
+2. Start server
 
+```bash
 # Create database user rgsoc with password rgsoc
-$ createuser -P -s rgsoc
+$ sudo -u postgres createuser -P -s rgsoc
 Enter password for new role: rgsoc
 Enter it again: rgsoc
 ```
-
-## Bootstrap
-
-Copy `config/database.yml.example` to `config/database.yml`. Then make sure you
+3. Copy `config/database.yml.example` to `config/database.yml`. Then make sure you
 modify the settings so it could connect to your postgres server.
 
```

Comment:
Maybe it would be clearer to point out that `config/database.yml` needs to be created? Something along the lines of,

Copy the contents of `config/database.yml/example` to a new file `config/database.yml`. You want to make sure you modify the settings and add correct credentials so it can connect to your postgres server (see below).

Would that work?